packages/nimble-components/src/table-column/mixins/tests/groupable-column.spec.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable max-classes-per-file */
+import {
+    ViewTemplate,
+    ElementStyles,
+    html,
+    customElement
+} from '@microsoft/fast-element';
+import {
+    fixture,
+    Fixture,
+    uniqueElementName
+} from '../../../utilities/tests/fixture';
+import type { TableCellState } from '../../base/types';
+import { TableColumn } from '../../base';
+import { mixinGroupableColumnAPI } from '../groupable-column';
+import { TableGroupHeaderView } from '../../../table/components/group-header-view';
+
+abstract class TestTableColumnBase extends TableColumn {
+    public cellTemplate: ViewTemplate<TableCellState> = html``;
+    public cellStyles?: ElementStyles | undefined;
+    public cellRecordFieldNames: readonly string[] = [];
+}
+
+const testColumnGroupHeaderName = uniqueElementName();
+@customElement({
+    name: testColumnGroupHeaderName
+})
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class TestTableColumnGroupHeader extends TableGroupHeaderView {}
+
+const columnName = uniqueElementName();
+@customElement({
+    name: columnName
+})
+class TestTableColumn extends mixinGroupableColumnAPI(TestTableColumnBase) {
+    public groupHeaderViewTag = testColumnGroupHeaderName;
+}
+
+// prettier-ignore
+async function setup(): Promise<Fixture<TestTableColumn>> {
+    return fixture(columnName);
+}
+
+describe('GroupableColumn', () => {
+    let element: TestTableColumn;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setup());
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('groupable column implementation sets groupHeaderViewTag', async () => {
+        await connect();
+        element.groupHeaderViewTag = testColumnGroupHeaderName;
```

Comment:
I think this was a test that was (at one time) trying to assert that we were updating the `internalGroupHeaderViewTag` in the implementation of the `GroupableColumn` mixin. This has since changed such that that isn't necessary. I've removed the test.